core/src/test/scala/integration/kafka/api/BaseQuotaTest.scala

```diff
@@ -168,9 +168,8 @@ abstract class QuotaTestClients(topic: String,
                                 val consumer: KafkaConsumer[Array[Byte], Array[Byte]]) {
 
   def userPrincipal: KafkaPrincipal
-  def overrideQuotas(producerQuota: Long, consumerQuota: Long, requestQuota: Double)
-  def removeQuotaOverrides()
-
```

Comment:
nit: ditto for the space. I would personally prefer if we put spaces in between the methods